ethereum/permissioning/src/main/java/tech/pegasys/pantheon/ethereum/permissioning/PermissioningConfigurationBuilder.java

```diff
@@ -77,25 +120,24 @@ public static LocalPermissioningConfiguration permissioningConfiguration(
         permissioningConfiguration.setAccountWhitelist(accountsWhitelistToml);
       } else {
         throw new Exception(
-            ACCOUNTS_WHITELIST + " config option missing in TOML config file " + configFilePath);
+            ACCOUNTS_WHITELIST_KEY
+                + " config option missing in TOML config file "
+                + accountPermissioningConfigFilepath);
       }
     }
 
-    if (permissionedNodeEnabled) {
-      if (nodeWhitelistTomlArray != null) {
-        List<URI> nodesWhitelistToml =
-            nodeWhitelistTomlArray
-                .toList()
-                .parallelStream()
-                .map(Object::toString)
-                .map(EnodeURL::asURI)
-                .collect(Collectors.toList());
-        permissioningConfiguration.setNodeWhitelist(nodesWhitelistToml);
-      } else {
-        throw new Exception(
-            NODES_WHITELIST + " config option missing in TOML config file " + configFilePath);
-      }
-    }
     return permissioningConfiguration;
   }
+
+  private static TomlParseResult readToml(final String filepath) throws Exception {
+    TomlParseResult nodePermissioningToml;
```

Comment:
I think this method is used for both node and account permissioning - can take 'node' out of variable name